storage/storage.cpp

```diff
@@ -601,9 +600,11 @@ void Storage::OnServerListDownloaded(vector<string> const & urls)
   TCountryId const & countryId = queuedCountry.GetCountryId();
   MapOptions const file = queuedCountry.GetCurrentFile();
 
+  vector<string> const downloadingUrls =
```

Comment:
> Do you really need a copy here, or const reference will be enough?

You're right. The const reference is enough. Fixed.

> BTW, why not to use auto const & here?

Yes, vector<string> is longer than auto. But in my opinion it's more clear while you read the code which type it is. I prefer with exchange with auto if "exchangee" is longer.